README.md

```diff
@@ -98,25 +168,33 @@ and
 <%= javascript_pack_tag 'shop/orders' %>
 ```
 
-## Advanced Configuration
+### Paths
 
 By default, webpacker offers simple conventions for where the webpack configs, javascript app files and compiled webpack bundles will go in your rails app,
 but all these options are configurable from `config/webpack/paths.yml` file.
 
+Suppose, you want
+to change the source directory from `app/javascript` to `frontend` and entry directory from `packs` to `modules` this is how you would do it.
+
 ```yml
 # config/webpack/paths.yml
-source: app/javascript
-entry: packs
+source: frontend
+entry: modules
 output: public
-config: config/webpack
-node_modules: node_modules
```

Comment:
This `paths.yml` example should be updated to reflect the changes in https://github.com/rails/webpacker/pull/397